MIT License を設定する (#57)
YNUCPC/ynu-icpc-library: README.md

    ...
    \input{docs/<hoge>/<fuga>.tex}
    ```

## License

YNU ICPC Library は、[MIT License](https://opensource.org/licenses/MIT) のもとで公開されています。
ただし、プログラミングコンテストや Online Judge のジャッジサーバーに提出する目的でこのライブラリを使用する場合は、提出物に著作権表示や許諾表示をすることなく、このライブラリを使用することができます。
ライセンス全文は [LICENSE](https://github.com/YNUCPC/ynu-icpc-library/blob/main/LICENSE) を参照してください。

YNU ICPC Library is licensed under the [MIT License](https://opensource.org/licenses/MIT).

> However, if you use this library for submission to the judge server of a programming contest or online judge, you can use this library without any copyright and permission notices on your submission.

Reviewer: ```suggestion
However, you can use this library for submission to the judge server of a programming contest or online judges without any copyright or permission notices on your submission.
```
 直訳からは外れますが、この方が簡潔になると思ったので提案です。

Author: ありがとうございます 🙇 
確かに簡潔になると思いましたので [bb12284](https://github.com/YNUCPC/ynu-icpc-library/pull/57/commits/bb12284b4914c3be4e78e811a72f213949bba976) で反映しました
